doc/source/parameters/dem/insertion_info.rst

```diff
@@ -53,38 +48,38 @@ In this subsection, insertion methods which are ``volume``, ``plane``, ``list``
+* ``insertion frequency`` defines the frequency of the insertion of particles in the box. For example, if the ``insertion frequency`` is set to 10000, the iterations 1, 10001, 20001, ... will be defined as insertion steps. The frequency should be large, so that the inserted particles at the previous insertion step have enough time to leave the insertion box for the next insertion step, otherwise large overlap may occur which leads to a large velocity of particles.
 
-* The ``insertion box points coordinates`` parameter defines the insertion box dimensions using two points: ``x1, y1, z1 : x2, y2, z2``. It is the same principle has what is being done for the `CFD <https://chaos-polymtl.github.io/lethe/documentation/parameters/cfd/mesh.html>`_ triangulation.
+* ``insertion maximum offset`` defines the maximum value of the offset in relation to the structured discrete positions in the box. The insertion locations of particles are randomly selected if the offset is not equal to zero, otherwise, the particles will perfectly aligns along the x, y and z directions.
 
-.. note::
-    We recommend that the defined insertion box have at least a distance of :math:`{d^{max}_p}` (maximum diameter of particles) from the triangulation boundaries. Otherwise, particles may have an overlap with the triangulation walls in the insertion.
+* ``insertion prn seed`` seeds the pseudo-random number (PRN) generator. It defines the starting value from which the offset values are generated.
+
+* ``inserted number of particles at each time step`` defines the desired number of particles to be inserted at each insertion step. If the insertion box is not adequately large to insert this number of particles with the defined arrangement (initial distance between the inserted particles), the maximum number of particles that fit inside the insertion box at each insertion step are inserted and a warning is displayed.
 
-* The ``insertion order of direction`` parameter defines the directions of insertion. For example, if the parameter is equal to ``0, 1, 2``, the particles are inserted in priority in the x, in y, and then in z directions. This is the default configuration. This is useful to specify the insertion directions to cover a specific area of the insertion box with the first and second direction parameters.
+* ``insertion box points coordinates`` defines the insertion box dimensions using two points: ``x1, y1, z1 : x2, y2, z2``. It is based on the same principle as for the rectangular mesh `CFD <../../parameters/cfd/mesh.html>`_ triangulation.
 
-* The ``initial velocity`` determine the initial translational velocity (in :math:`\frac{m}{s}`) at which particles are inserted in the x, y, and z directions.
+.. note::
+    We recommend that the defined insertion box has at least a distance of :math:`{d^{max}_p}` (maximum diameter of particles) from the triangulation boundaries. Otherwise, particles may have an overlap with the triangulation walls in the insertion.
```

Comment:
maximum diameter of particle plus the maximum offset maybe?

Reply:
Good point. I have a question related to that. I think when the insertion method were uniform vs non uniform, the discretization in the box taking that into account, but I did not managed to find it in the current code. Was it removed? If so, would it be a good idea to add it since we have to do it?

Reply:
Ok I managed to find where the code does take that into account. Actually, it works if there are no offset (according to function `find_insertion_location_volume()` in insertion_volume.cc), the minimal position will be the max distance threshold * radius. Since the threshold should be always > 1, we should not take into account the Dp,max. The code should automatically take the threshold into account tho!
I will add the maths in the doc

![image](https://github.com/user-attachments/assets/dd7f527c-53a5-4ff5-9ca3-e1133f977781)

Reply:
I tested the parameters and find out a few criteria to avoid overlap. I did add them in the doc.